Python/pystate.c

```diff
@@ -370,6 +371,7 @@ PyInterpreterState_New(void)
         // Set to _PyInterpreterState_INIT.
         memcpy(interp, &initial._main_interpreter,
                sizeof(*interp));
+        interp->_static = false;
```

Comment:
Likewise, a note pointing to [`PyInterpreterState_INIT()`](https://github.com/python/cpython/blob/2eee9d9cd7eb1e396fa9a4af7c5fadeeafbdaa38/Include/internal/pycore_runtime_init.h#L48-L66).